### ISSUE TITLE
Toggle - Not able to disable 'default wallet' toggle

### ISSUE THREAD

**Issue by mr-michael** [tallycash/extension#669]
tl;dr the wallet works perfectly at first. Then if I disable 'default wallet' toggle, the toggle displays as 'on' and I'm not able to maintain normal dapp connections on uniswap, yearn, or curve.

Deets:

I have MM installed (and unlocked) but never open it during this time.

I install and open Tally and leave default wallet enabled (I do not open Menu or touch the setting). I can log into UNI, curve, yearn. It works great!

Then I disable toggle. I then view my Wallet and go back to Menu and the toggle has re-enabled itself.

I go to curve and yearn and the Tally option is no longer visible (previously curve showed a Tally option and Yearn showed a 'loaded wallet' (or something) option):

<img width="717" alt="Screen Shot 2021-12-15 at 2 47 00 PM" src="https://user-images.githubusercontent.com/7005061/146256170-ef72fdc4-43e4-4a60-b116-6e378995acd0.png">

<img width="756" alt="Screen Shot 2021-12-15 at 2 47 16 PM" src="https://user-images.githubusercontent.com/7005061/146256172-c0f0d8cd-0d27-452c-a093-d81640837845.png">

I go to Uniswap and the UNI UI shows that I am persistently connected (even after multiple hard refreshes), but in Tally it doesn't display that I am connected to uniswap (no lightning icon appears). You can see in this image below (barely) on the uniswap UI that I'm still connected (note: I checked and am not connected via MM):

<img width="485" alt="Screen Shot 2021-12-15 at 2 51 06 PM" src="https://user-images.githubusercontent.com/7005061/146256269-c824f011-3415-4f27-a09a-5c1664d2702c.png">

It stays like this until I reinstall Tally.

**Comment by greg-nagy** [comment]
This is fixed with: https://github.com/tallycash/extension/pull/620

Not closing so we can double check when the PR has landed.

**Comment by greg-nagy** [comment]
actually closing it. let's reopen if it is not working
